### ISSUE TITLE
DB creation throws error about vnic_id but completes build

### ISSUE THREAD

**Issue by ramnathnayakpr** [oracle/terraform-provider-baremetal#149]
## Description of the bug:
The DBSystem example creates the DB, but throws this error towards the end:
Error applying plan:

```
1 error(s) occurred:

* data.baremetal_core_vnic.DBNodeVnic: Resource 'data.baremetal_database_db_node.DBNodeDetails' does not have attribute 'vnic_id' for variable 'data.baremetal_database_db_node.DBNodeDetails.vnic_id'

```

## A self-contained terraform file that reproduces the issue:

[DBsystems.zip](https://github.com/oracle/terraform-provider-baremetal/files/1078173/DBsystems.zip)

(If this is not provided, your ticket will be closed)

**Comment by mikejihbe** [comment]
This looks like user error. This example is passing a `baremetal_database_db_system.<name>.id` into a db_node_id parameter. You'd have to pass in a node_id for this to work.

**Comment by ramnathnayakpr** [comment]
As this is an issue with the example and not the terraform provider, should this issue be closed?

**Comment by codycushing** [comment]
Fixed in v1.0.15
